neqo-transport/src/packet/mod.rs

```diff
@@ -128,13 +128,15 @@ pub struct PacketBuilder {
     padding: bool,
 }
 
-impl PacketBuilder {
+impl<'a> PacketBuilder<'a> {
     /// The minimum useful frame size.  If space is less than this, we will claim to be full.
     pub const MINIMUM_FRAME_SIZE: usize = 2;
 
-    fn infer_limit(encoder: &Encoder) -> usize {
-        if encoder.capacity() > 64 {
-            encoder.capacity()
+    const fn infer_limit(limit: usize) -> usize {
+        // TODO: I don't know what the 64 is all about. Thus leaving the infer_limit function intact
+        // for now.
```

Comment:
Huh. @martinthomson might know.

Reply:
For what it is worth, the value `64` was introduced in https://github.com/mozilla/neqo/pull/998.

Also note that no unit test depends on the value `64`, in other words, removing it does not cause any test to fail.

Reply:
Ping @martinthomson 

We should also maybe make `64` and `2048` appropriately-named `const`s, if we determine what they do & that we want to keep them.

Reply:
The value exists to ensure that we respect the capacity when it is already set.  I forget the details.